lib/openxml/pptx/properties/placeholder.rb

```diff
@@ -0,0 +1,27 @@
+module OpenXml
+  module Pptx
+    module Properties
+      class Placeholder < OpenXml::Properties::ComplexProperty
+        include OpenXml::ContainsProperties
+        namespace :p
+        tag :ph
+
+        # ST_PlaceholderType - Part 1, 19.7.10
+        PLACEHOLDER_TYPES = %i{ title body ctrTitle subTitle dt sldNum ftr hdr obj
+                                chart tbl clipArt dgm media sldImg pic }.freeze
+
+        # ST_PlaceholderSize - Part 1, 19.7.9
+        PLACEHOLDER_SIZES = %i{ full half quarter }.freeze
```

Comment:
Extract to `OpenXml::Pptx::ST_PlaceholderType` and `OpenXml::Pptx::ST_PlaceholderSize`?